src/plus/startWork/startWork.ts

```diff
@@ -166,6 +172,10 @@ export class StartWorkCommand extends QuickCommand<State> {
 					}
 				}
 
+				if (!isStateWithType(state)) {
+					continue;
+				}
```

Comment:
We should probably throw here and possibly include a `debugger` line if we don't have a `type` by this point, because something must have gone very wrong. Maybe we can use a function similar to `assertsLaunchpadStepState`?

Reply:
@axosoft-ramint fixed